### ISSUE TITLE
When clear chat is pressed two times, subsequent requests don't generate anything

### ISSUE THREAD

**Issue by VictorZakharov** [ItsPi3141/alpaca-electron#25]
**Describe the bug**
When clear chat is pressed two times, subsequent requests don't generate anything.

**To Reproduce**
1. Ask something. Wait for bot to answer. Observe CPU is around 70%. I have a Ryzen 9 5900x.
2. Press clear chat twice.
3. Ask something else.
4. Nothing happens, CPU is idle.

**Expected behavior**
Should be able to continue in a new conversation without having to end process in Task Manager every time.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Windows 10.
 - Browser: Chrome.
 - Version: 112

**Additional context**
Using Alpaca 13B native model linked from the Releases page.

**Comment by ItsPi3141** [comment]
Yes that's a known issue with llama.cpp itself. I will try to implement a temporary fix for it soon.
